src/components/shared/OutlinedToggleButtonGroup.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+    styled,
+    ToggleButtonGroup,
+    ToggleButtonGroupProps,
+} from '@mui/material';
+
+const OutlinedToggleButtonGroup = styled(ToggleButtonGroup)<
+    ToggleButtonGroupProps & { buttonSelector?: string }
+>(({ buttonSelector }) => ({
+    [`&${buttonSelector ?? ' .MuiToggleButton-root'}`]: {
```

Comment:
I am thinking that if we allow a `selector` to be passed in then the person doing it needs to handle nesting themselves. I would say requiring an entire string is best.

So the `&` should be passed in.